src/components/pages/LoginPage.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { push } from 'connected-react-router';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import { VerticalSpacer } from 'components/atoms/Spacer';
+import SolidIconButton from 'components/atoms/SolidIconButton';
+import { useDispatch } from 'react-redux';
+import { Formik, FormikProps } from 'formik';
+import { bps } from 'theme';
+import FormikTextField from 'components/atoms/FormikTextField';
+import DoneButton from '../atoms/DoneButton';
+import { updateToken } from '../../states/auth/actions';
+import { PATH_APP } from '../../libraries/constants';
+
+
+const StyledDiv = styled.div`
+  box-sizing: border-box;
+  padding-top: 35%;
+  padding-left: 40px;
+  padding-right: 40px;
+  ${bps.down('sm')} {
+    padding: 40% 20px 0 20px;
+  }
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  .retry-button {
+    width: 140px;
+  }
+`;
+
+interface LoginProps {
+  secret: string;
+}
+
+/**
+ * This page will be revamped soon. Once we have a proper custom error page it should not be named maintenance anymore
```

Comment:
Got it

Reply:
lol that was the code i copied from somewhere else